src/schema/generated-schema-types.ts

```diff
@@ -129,6 +129,7 @@ export type AccountInfoAttrs = {
   zimbraMtaMaxMessageSize?: Maybe<Scalars['Float']>;
   zimbraPasswordAllowedChars?: Maybe<Scalars['String']>;
   zimbraPasswordAllowedPunctuationChars?: Maybe<Scalars['String']>;
+	zimbraPasswordAllowUsername?: Maybe<Scalars['Boolean']>;
```

Comment:
is this manually added? this shouldn't have indentation issue as this is auto generated file

Reply:
updated the file again with correct auto generation